account_document_reversal/models/account_move.py

```diff
@@ -63,10 +63,10 @@ def action_document_reversal(self, date=None, journal_id=None):
                 )
             )
         # Create reverse entries
-        self._cancel_reversal(journal_id)
+        self._cancel_reversal(journal_id, date=date)
         return True
 
-    def _cancel_reversal(self, journal_id):
+    def _cancel_reversal(self, journal_id, date=None):
```

Comment:
```suggestion
    def _cancel_reversal(self, journal_id, date):
```
Are there any case when date is not assigned, can you make sure? If not the case, we may not need `if date`, possible?

Reply:
Yes, date is required on wizard. I make params optional only for not change function's signature, otherwise if any module custom inherit this function raise error

So, if it is not problem, i change params as your suggestion